src/main/java/mate/academy/spring/dto/request/MovieRequestDto.java

```diff
@@ -1,7 +1,17 @@
 package mate.academy.spring.dto.request;
 
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+
 public class MovieRequestDto {
+    @NotNull
+    @NotBlank(message = " field can't be empty")
```

Comment:
```suggestion
    @NotBlank(message = "field can't be empty")
```
I believe you don't need spaces in the annotation messages